### PR TITLE
Removed unnecessary files

### PR DESCRIPTION
**Pull request by reynard2007**
Task #12

### CODE REVIEW

**Comment by vikasrohit** [comment]
@reynard2007 you can start resolving issues for this now.

File: components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+'use strict'
+
```
Comment (nlitwin) [inline components/Navbar/Navbar.jsx]: same as 'use strict' / imports comments here https://github.com/appirio-tech/react-components/commit/3118fed8f68e8ca4a511142d2cbdf982f9344a6d

File: components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const primaryNavigationItems = [
+  {img: '../components/MenuBar/nav-community.svg', text: 'Community', link: 'javascript:;'},
+  {img: '../components/MenuBar/nav-compete.svg', text: 'Compete', link: 'javascript:;', selected: true},
+  {img: '../components/MenuBar/nav-learn.svg', text: 'Learn', link: 'javascript:;'}
+]
+
+const Navbar = {
+  render() {
+    const dom = (
```
Comment (nlitwin) [inline components/Navbar/Navbar.jsx]: Return jsx code directly (no `dom` var)

File: components/Navbar/Navbar.scss [hunk 3]
```diff
@@ -0,0 +1,121 @@
+@import "work/work-includes";
```
Comment (nlitwin) [inline components/Navbar/Navbar.scss]: Use `topcoder/tc-includes` instead

File: components/Navbar/NavbarExample.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+'use strict'
+
```
Comment (nlitwin) [inline components/Navbar/NavbarExample.jsx]: Same comments as above

**Comment by nlitwin** [comment]
The navbar example isn't rendering correctly for me. 
image

**Comment by nlitwin** [comment]
Can we fix the width of the searchbar? Also the quicklinks and user dropdown don't have `cursor: pointer`

**Comment by reynard2007** [comment]
These are acknowledged. Will create a new PR for the above and the other requests for updates within the day (PHT).

**Comment by reynard2007** [comment]
Closing. To be replaced by PR #37.

**Pull request closed without merging**